src/averageAges.js

```diff
@@ -14,7 +14,26 @@
  *
  * @return {number}
  */
+function calculateAge(array) {
+  return array.reduce((acc, person) => {
+    const age = person.died - person.born;
+
+    return acc + age;
+  }, 0) / array.length;
+}
+
 function calculateMenAverageAge(people, century) {
+  const filterMen = century
+    ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
```

Comment:
You could just pass different callbacks to the filter method depending on the condition.
You can name just men in this case.
Also, let's create variables for sex values such as "m", "f"